CMakeLists.txt

```diff
@@ -47,9 +52,12 @@ add_executable(test_abieos src/test.cpp src/abieos.cpp)
 target_link_libraries(test_abieos abieos ${CMAKE_THREAD_LIBS_INIT})
 add_test(NAME test_abieos COMMAND test_abieos)
 
-add_executable(test_abieos_template src/template_test.cpp src/abieos.cpp)
-target_link_libraries(test_abieos_template abieos ${CMAKE_THREAD_LIBS_INIT})
-add_test(NAME test_abieos_template COMMAND test_abieos_template)
+if(ABIEOS_NO_INT128)
```

Comment:
if(NOT ABIEOS_NO_INT128)?

Reply:
I wish. cmake bombed with that.

Reply:
It didn't work before but now it does. weird. changed.